src/analyzer/protocol/mime/MIME.cc

```diff
@@ -1131,7 +1131,8 @@ void MIME_Entity::DecodeBase64(int len, const char* data)
 		{
 		rlen = 128;
 		char* prbuf = rbuf;
-		int decoded = base64_decoder->Decode(len, data, &rlen, &prbuf);
+		size_t x = static_cast<size_t> (rlen);
+		int decoded = base64_decoder->Decode(len, data, &x, &prbuf); // typecasting for now
```

Comment:
I am fixing this in the merge. But note - this leads to wrong results. x is changed when it is passed to decode - and your re-assignment shadows this and does not reflect the change in rlen.